frontend/src/admin/categories/CategoryDetails.tsx

```diff
@@ -71,9 +68,11 @@ export default function CategoryDetails() {
               Are you sure you want to delete this category?
             </h1>
           </Modal>
-          {state.isLoading && !state.isError && <Loader />}
-          {state.isError && !state.isLoading && <h1>error</h1>}
-          {!state.isError && !state.isLoading && (
+          {state.category.isLoading && !state.category.isError && <Loader />}
+          {state.category.isError && !state.category.isLoading && (
+            <h1>error</h1>
```

Comment:
Dont use h1. Please check rest of your elements. avoid using header elements for these kind of info.
Just make use of `div` or `span`